maven/lib/dependabot/maven/file_parser/repositories_finder.rb

```diff
@@ -1,4 +1,4 @@
-# typed: false
+# typed: true
```

Comment:
should this be `strict`?

Reply:
I suspect this was `false` because something in `MetadataFinder` was causing a Sorbet error. But now `MetadataFinder` is `strict` typed, it's no longer an error.

Personally, I don't mind if it's made strict in this PR or another one. But it needs to be a least `true` for CI to pass.